packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-header.jsx

```diff
@@ -38,6 +40,12 @@ const VerticalTabHeader = ({ children, className, is_routed, item, onChange, sel
             >
                 <HeaderIcon icon={item.icon} is_active={is_active} />
                 <Header text={label} />
+                {!!count &&
+                    <Counter
+                        count={count}
+                        className='vertical-tab__header__counter'
```

Comment:
have to add styling to vertical-tab__header__counter scss for margin left

Reply:
It has it already in `vertical-tabs.scss` (line: 109)